clearing-house/src/tests/open_position.rs

```diff
@@ -16,14 +16,14 @@ use crate::{
         any_direction, any_price, as_balance, get_collateral, get_market, get_market_fee_pool,
         get_outstanding_profits, get_position, run_for_seconds, run_to_time, set_fee_pool_depth,
         set_maximum_oracle_mark_divergence, set_oracle_price, set_oracle_twap,
-        with_markets_context, with_trading_context, Market, MarketConfig,
+        with_markets_context, with_trading_context, Market, MarketConfig, comp::{approx_eq_lower, approx_eq},
```

Comment:
Just a NIT: maybe there would be a better name for the file `comp.rs`? I don't think it's obvious that `comp` is short for `comparison` (it initially gave me the idea of 'computation'). Maybe a naming along the lines of `tolerance`, `approximations`, `check_diff` or `comparison` would be better?

Just an idea really, up to you to go with a change or not.